views/layout.njk

```diff
@@ -5,7 +5,8 @@
       window._serverData = {
         domain: '{{ domain }}',
         name: '{{ name }}',
-        hub: '{{ hub }}',
+        hub: '{{ hub[0] }}',
+        cors: '{{ hub.join() }}'
```

Comment:
this line is missing a comma, but we don't need this info in the web views anyway. The only thing `hub` is used for on the front end appears to be the [start button link](https://github.com/immers-space/immers/blob/master/views/components/Layout.js#L43), so might be even better to just change the renderConfig in index.js to only pass `hub[0]` down to the view in the first place

![image](https://user-images.githubusercontent.com/10034859/166834975-222935fa-1546-4e5d-a1d0-f6a2908333c1.png)